theme.toml

```diff
@@ -60,6 +60,11 @@ error = "apricot"
 info = "delta"
 hint = "silver"
 
+diff.added = "#35BF86"
```

Comment:
Okay so this is why: the color names aren't quoted so they parse as nested keys. Any key with a dot in the name needs to be quoted.